### PR TITLE
BZ2020300: Importer pod log unavailable after warm migration

### PR DESCRIPTION
**Pull request by apinnick**
https://bugzilla.redhat.com/show_bug.cgi?id=2020300

Preview: https://deploy-preview-243--forklift-documentation.netlify.app/documentation/doc-release_notes/master/#known-issues-22_forklift

Ready for QE.

### CODE REVIEW

**Comment by apinnick** [comment]
@mguetta1 I updated the PR. Ready for review.

File: documentation/modules/rn-2.2.adoc [hunk 1]
```diff
@@ -31,3 +31,13 @@ The release notes describe new features and enhancements, known issues, and tech
 .QEMU guest agent is not installed on migrated VMs
 
 The QEMU guest agent is not installed on migrated VMs. Workaround: Install the QEMU guest agent with a post-migration hook. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2018062[*BZ#2018062*])
+
+.Importer pod log is unavailable after warm migration
+
+Retaining the `Importer` pod causes warm migration to hang during the cutover stage. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2016290[*BZ#2016290*])
```
Comment (mguetta1) [inline documentation/modules/rn-2.2.adoc:37]: ...during precopy stage
https://bugzilla.redhat.com/show_bug.cgi?id=2016290#c14

File: documentation/modules/rn-2.2.adoc [hunk 2]
```diff
@@ -31,3 +31,13 @@ The release notes describe new features and enhancements, known issues, and tech
...
+
+As a workaround, the `Importer` pod is removed at the end of the precopy stage so that the cutover stage succeeds. However, as a result, the `Importer` log is unavailable after the warm migration is completed. If you download logs by using the web console or the `must-gather` tool, the `Importer` log is not included. The `virt-v2v` logs are available, however.
```
Comment (mguetta1) [inline documentation/modules/rn-2.2.adoc:39]: so the precopy stage succeeds

**Review by mguetta1 — CHANGES_REQUESTED** [review]

**Review by mguetta1 — APPROVED** [review]
LGTM

**Pull request merged**